gluoncv/model_zoo/model_zoo.py

```diff
@@ -54,6 +54,7 @@
     'resnet50_v2': resnet50_v2,
     'resnet101_v2': resnet101_v2,
     'resnet152_v2': resnet152_v2,
+    'resnest26': resnest26,
```

Comment:
Could you please share the context for adding this model?

Reply:
this is a lightweight model for resnest.